includes/html/print-map.inc.php

```diff
@@ -315,10 +334,25 @@
 $nodes = json_encode(array_values($devices_by_id));
 $edges = json_encode($links);
 
+array_multisort(array_column($devices_by_id, 'label'), SORT_ASC, $devices_by_id);
+
 if (count($devices_by_id) > 1 && count($links) > 0) {
 ?>
 
```

Comment:
Maybe it should be a GET request so you can link to it

Reply:
@Jellyfrog 
good idea, i'll change to it.
I'll adapt this to #10916 also, if it's merged :-)

Reply:
@Jellyfrog 
on select box change URL not changing.
But on setting: `/map/highlight_node=<device_id>` Node is highlighted